initiateContainerTest.sh

```diff
@@ -21,6 +21,8 @@ CWD=`pwd`
 echo "Running in $CWD, setting up envtest."
 
 export ENVTEST_ASSETS_DIR=/nnf/testbin
+export GOMEGA_DEFAULT_EVENTUALLY_TIMEOUT=20s
+export GOMEGA_DEFAULT_EVENTUALLY_INTERVAL=100ms
```

Comment:
Given the volume of output from this particular test suite, I'd like to see the following added to the "go test" commandline below:  `-args -ginkgo.v -ginkgo.progress -ginkgo.failFast`
We use that in the  makefile.

Reply:
Bah, I didn't mean to click approve with that...